src/checks/json.check.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 const status = require('./status');
 const Check = require('./check');
+const fetch = require('node-fetch');
```

Comment:
although fyi this will mean that requests going through here won't be instrumented and we won't get graphite data for them